code/game/atoms.dm

```diff
@@ -1415,6 +1415,23 @@
 	if(source != target)
 		target.log_talk(message, message_type, tag="[tag] from [key_name(source)]", log_globally=FALSE)
 
+/**
+  * Log for crafting items
+  *
+  * 1 argument is for the user making the item
+  * 2 argument is for the item being created
+  * 3 argument is for if admins should be notified if a non-antag crafts this
+ */
+/proc/log_crafting(atom/user, atom/object, dangerous = FALSE)
+	var/masterpiece = object
```

Comment:
why does this var even exist, it's a straight copy from `object` and isn't changed